manifest.md

```diff
@@ -180,8 +189,16 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
     }
   ],
   "annotations": {
-    "key1": "value1",
-    "key2": "value2"
+    "com.example.key1": "value1",
+    "com.example.key2": "value2"
   }
 }
 ```
+
+# Pre-Defined Annotation Keys
```

Comment:
This DRYs up the pre-defined annotation keys, which is good.  But there's still a lot of duplicate language about extensibility and the `annotations` field itself.  Can we [collect *all* of that into one location and link to it from other consumers][1]?

[1]: https://github.com/opencontainers/image-spec/pull/164#issuecomment-243952187